src/lsp/server/utils/serverWorkspace.ts

```diff
@@ -5,35 +5,20 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-import { SFDX_PROJECT_FILE } from './constants';
-import * as fs from 'fs';
-import * as path from 'path';
 import { WorkspaceFolder } from 'vscode-languageserver';
 
-export class WorkspaceUtils {
-    // WorkspaceFolders is null if Workspace is not configured
+export class ServerWorkspace {
     private static workspaceFolders: WorkspaceFolder[] | null;
 
-    static readonly DEFAULT_APP_PATH = path.join(
-        'force-app',
-        'main',
-        'default'
-    );
-
-    static readonly LWC_PATH = path.join(
-        WorkspaceUtils.DEFAULT_APP_PATH,
-        'lwc'
-    );
-
     /**
      * WorkspaceFolders must be initialized before they can be used.
      * @param workspaceFolders workspaceFolders passed from LSP server connection.
      */
-    public static initWorkspaceFolders(
+    static initWorkspaceFolders(
         workspaceFolders: WorkspaceFolder[] | null | undefined
     ) {
         if (workspaceFolders === undefined) {
-            throw new Error('LSP client does no support workspace folder');
+            throw new Error('LSP client requires workspace folder.');
```

Comment:
Another reason why the name `ServerWorkspace` may not be the best option..... The class/file name is `ServerWorkspace` but the error message references `client`

Reply:
@haifeng-li-at-salesforce might be able to answer this better as I'm leaving much of code written originally for LSP's first implementation. If I were to attempt answering it, this method is ONLY being called in LSP server as it is getting bootstrapped. That bootstrap process is started by client. And that's why it probably has reference to `client`.